webpack/JobWizard/steps/AdvancedFields/__tests__/AdvancedFields.test.js

```diff
@@ -72,7 +64,7 @@ describe('AdvancedFields', () => {
       .simulate('click');
 
     expect(wrapper.find('.pf-c-wizard__nav-link.pf-m-current').text()).toEqual(
-      'Target Hosts'
+      'Target hosts and inputs'
```

Comment:
We usually dont use consts in tests. I think this could cause issues if there is an issue with the constant, for example if it's undefined or has a typo.